apps/cowswap-frontend/src/modules/volumeFee/updaters/TaxFreeAssetsUpdater.tsx

```diff
@@ -0,0 +1,106 @@
+import { useSetAtom } from 'jotai'
+
+import { getCmsClient } from '@cowprotocol/core'
+import { mapSupportedNetworks, SupportedChainId } from '@cowprotocol/cow-sdk'
+
+import ms from 'ms.macro'
+import qs from 'qs'
+import useSWR, { SWRConfiguration } from 'swr'
+
+import { taxFreeAssetsAtom } from '../state/taxFreeAssetsAtom'
+
+type TokenId = string
+
+type TaxFreeAssetItem = {
+  attributes: {
+    tokenIds: string
+    chainId: {
+      data: {
+        attributes: {
+          chainId: number
+        }
+      }
+    }
+  }
+}
+
+const UPDATE_INTERVAL = ms`10m`
+
+const SWR_CONFIG: SWRConfiguration = {
+  refreshInterval: UPDATE_INTERVAL,
+  revalidateOnFocus: false,
+}
+
+const UPDATE_TIME_KEY = 'taxFreeAssetsUpdateTime'
+
+const cmsClient = getCmsClient()
+
+const querySerializer = (params: any) => {
+  return qs.stringify(params, { encodeValuesOnly: true, arrayFormat: 'brackets' })
+}
+
+export function TaxFreeAssetsUpdater() {
+  const setTaxFreeAssets = useSetAtom(taxFreeAssetsAtom)
+
+  useSWR(
+    ['/tax-free-assets', setTaxFreeAssets],
+    async ([method, setTaxFreeAssets]) => {
+      const lastUpdateTime = localStorage.getItem(UPDATE_TIME_KEY)
```

Comment:
Nitpick: use an atom with storage instead?

Reply:
Same as here: https://github.com/cowprotocol/cowswap/pull/5200#issuecomment-2541313163

But I will try to use `getJotaiMergerStorage `

Reply:
Unfortunately `getJotaiMergerStorage` doesn't help